modules/distribution-zones/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
     implementation project(':ignite-metastorage')
     implementation project(':ignite-vault')
     implementation project(':ignite-configuration')
+    implementation project(':ignite-schema')
```

Comment:
Why? Do you really mean that zone depend on schemas now?

Reply:
Because DistributionZoneManager has a TablesConfiguration which is located in 'ignite-schema'